cmd/daserver/daserver.go

```diff
@@ -59,23 +56,12 @@ func parseDAServer(args []string) (*DAServerConfig, error) {
 		return nil, err
 	}
 	if serverConfig.ConfConfig.Dump {
-		// Print out current configuration
-
-		// Don't keep printing configuration file
-		err := k.Load(confmap.Provider(map[string]interface{}{
-			"conf.dump": false,
-		}, "."), nil)
+		err = util.DumpConfig(k, map[string]interface{}{
+			"data-availability.local-disk.priv-key": "",
```

Comment:
Should be `data-availability.priv-key` since s3 also needs private key.  Best not to duplicate shared settings